src/game/items/CItemBase.cpp

```diff
@@ -1032,7 +1032,7 @@ int CItemBase::GetMakeValue( int iQualityLevel )
 
 	CValueRangeDef values(m_values);
 
-	if ( m_values.m_iLo == INT64_MIN || m_values.m_iHi == INT64_MIN )
+	if ( m_values.m_iLo == INT32_MIN || m_values.m_iHi == INT32_MIN )
```

Comment:
This line make my server crash on boot when i'm in debug or on Nightly

![image](https://github.com/user-attachments/assets/882502b0-928e-4036-84ab-60d1c593c305)